### PR TITLE
[Sunbeam GA Launch] - /openstack revamp

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated page as per [doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-12935.demos.haus/openstack
    - Be sure to test on mobile, tablet and desktop screen sizes
- Make sure copy changes are correct
- Check "Get your OpenStack" functionality ("I will manage OpenStack" should reveal 2 cards in step 2)
- Check carousel (logos needed from visual, will update once they are finalized)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3628

### CODE REVIEW

**Comment by juanruitina** [comment]
Nice job on the cards. Just a couple of comments:

- Please update the title and meta description of the page as per the copy doc.
- Deferring to @lyubomir-popov on how the logos should look like on the carousel; as long as the text in the logos is somewhat readable (it isn't know), I'll be fine with them.

**Comment by lyubomir-popov** [comment]
> * Deferring to @lyubomir-popov on how the logos should look like on the carousel; as long as the text in the logos is somewhat readable (it isn't know), I'll be fine with them.

It's just a matter of ordering them nicely:

image

```
<div class="p-logo-section__items">
        <div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_288/https://assets.ubuntu.com/v1/7eebe11c-cisco-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_576,h_576/https://assets.ubuntu.com/v1/7eebe11c-cisco-logo.png 2x" alt="Cisco" width="288" height="288" loading="lazy" class="p-logo-section__logo">
  </div>

        </div><div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_288/https://assets.ubuntu.com/v1/ea3bc2f7-telefonica-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_576,h_576/https://assets.ubuntu.com/v1/ea3bc2f7-telefonica-logo.png 2x" alt="Telefónica" width="288" height="288" loading="lazy" class="p-logo-section__logo">
  </div>

        </div>
        <div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_288/https://assets.ubuntu.com/v1/aed42949-aci-universal-payments-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_576,h_576/https://assets.ubuntu.com/v1/aed42949-aci-universal-payments-logo.png 2x" alt="ACI" width="288" height="288" loading="lazy" class="p-logo-section__logo">
  </div>

        </div>
        
        
        <div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_288/https://assets.ubuntu.com/v1/b1e3d2d2-bloomberg-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_576,h_576/https://assets.ubuntu.com/v1/b1e3d2d2-bloomberg-logo.png 2x" alt="Bloomberg" width="288" height="288" loading="lazy" class="p-logo-section__logo">
  </div>

        </div>
        <div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_288/https://assets.ubuntu.com/v1/bd2a3282-bt-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_576,h_576/https://assets.ubuntu.com/v1/bd2a3282-bt-logo.png 2x" alt="BT" width="288" height="288" loading="lazy" class="p-logo-section__logo">
  </div>

        </div>
        <div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_288/https://assets.ubuntu.com/v1/5a238b8f-gendarmerie-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_576,h_576/https://assets.ubuntu.com/v1/5a238b8f-gendarmerie-logo.png 2x" alt="Gendermarie" width="288" height="288" loading="lazy" class="p-logo-section__logo">
  </div>

        </div><div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_502,h_132/https://assets.ubuntu.com/v1/dfcb8df1-Kadokawa.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_1004,h_264/https://assets.ubuntu.com/v1/dfcb8df1-Kadokawa.png 2x" alt="Kadokawa Connected" width="502" height="132" loading="lazy" class="p-logo-section__logo">
  </div>

        </div><div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_288/https://assets.ubuntu.com/v1/698f7a58-best-buy-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_576,h_576/https://assets.ubuntu.com/v1/698f7a58-best-buy-logo.png 2x" alt="Best Buy" width="288" height="288" loading="lazy" class="p-logo-section__logo">
  </div>

        </div><div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_288/https://assets.ubuntu.com/v1/2b6c1718-bmp-paribas-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_576,h_576/https://assets.ubuntu.com/v1/2b6c1718-bmp-paribas-logo.png 2x" alt="BNP Paribas" width="288" height="288" loading="lazy" class="p-logo-section__logo">
  </div>

        </div>
        
        
        <div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_288/https://assets.ubuntu.com/v1/d858772a-spectrum-health-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_576,h_576/https://assets.ubuntu.com/v1/d858772a-spectrum-health-logo.png 2x" alt="Spectrum Health" width="288" height="288" loading="lazy" class="p-logo-section__logo">
  </div>

        </div>
        <div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_288/https://assets.ubuntu.com/v1/83fb74ae-nec-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_576,h_576/https://assets.ubuntu.com/v1/83fb74ae-nec-logo.png 2x" alt="NEC" width="288" height="288" loading="lazy" class="p-logo-section__logo">
  </div>

        </div><div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_1200,h_630/https://assets.ubuntu.com/v1/dda24c61-Nayatel.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_2400,h_1260/https://assets.ubuntu.com/v1/dda24c61-Nayatel.png 2x" alt="Nayatel" width="1200" height="630" loading="lazy" class="p-logo-section__logo">
  </div>

        </div>
        
        
        <div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_288/https://assets.ubuntu.com/v1/f0ba5797-t-systems-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_576,h_576/https://assets.ubuntu.com/v1/f0ba5797-t-systems-logo.png 2x" alt="T Systems" width="288" height="288" loading="lazy" class="p-logo-section__logo">
  </div>

        </div><div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_288/https://assets.ubuntu.com/v1/c2d36980-radobank+logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_576,h_576/https://assets.ubuntu.com/v1/c2d36980-radobank+logo.png 2x" alt="Robobank" width="288" height="288" loading="lazy" class="p-logo-section__logo">
  </div>

        </div><div class="p-logo-section__item">
          
<div class=" lazyloaded" data-noscript="">
    <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_288/https://assets.ubuntu.com/v1/a3bb7ca4-mercedes-benz-group-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_576,h_576/https://assets.ubuntu.com/v1/a3bb7ca4-mercedes-benz-group-logo.png 2x" alt="Mercedes Benz Group" width="288" height="288" loading="lazy" class="p-logo-section__logo">
  </div>

        </div>
        
      </div>
```

**Comment by mtruj013** [comment]
@lyubomir-popov that is not the section I need the logos for, this is:
image

Please see relevant conversation [here](https://warthogs.atlassian.net/browse/WD-3628?focusedCommentId=256552).

**Comment by lyubomir-popov** [comment]
> @lyubomir-popov that is not the section I need the logos for, this is:

Oh sorry, misunderstood. Can we still revise the logo section anyway - with the code I pasted?

@egezeytun can we get a freelancer to recise the logos for those testimonials?

**Comment by petesfrench** [comment]
Is this chart broken? No matter how you adjust VM hours, the bars in 'Private cloud' don't move. file:///home/pete/Videos/Screencasts/Screencast%20from%2012-06-23%2011:55:44.webm
-46:-33:84:67

**Comment by petesfrench** [comment]
[This line](https://github.com/canonical/ubuntu.com/pull/12935/files#diff-7565de9e0ec9a9e9299dd156a8024f89aeaabb631ef054d4e6ce28f63f14a128R412) should read "and _a_ fully-managed service" according to the doc

**Comment by lyubomir-popov** [comment]
> It sounds like we're all leaning toward removing the logos from the carousel for now, so I've gone ahead and done that, could you take a look and let me know if there's anything else you'd like me to change?

That's ok with me.

**Review by petesfrench — COMMENTED** [review]
looks good, just some comments minor in the code and some differences in the doc

File: templates/openstack/_testimonials.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+<script src="{{ versioned_static('js/dist/flickity.pkgd.min.js') }}"></script>
+
+<script>
+  initCarousel('#testimonials', '#testimonials-next', '#testimonials-previous');
+
+  function initCarousel(carouselSelector, nextCTAselector, previousCTAselector) {
+    var nextCTA = document.querySelector(nextCTAselector);
+    var previousCTA = document.querySelector(previousCTAselector);
+    var carousel = new Flickity(carouselSelector, {
+      imagesLoaded: true,
```
[the remark this answers is not in the document]
Comment (mtruj013) [reply templates/openstack/_testimonials.html]: Removed, there originally were but they need to be resized. Left a comment in the code to add this back in when available

File: templates/openstack/index.html [hunk 2]
```diff
@@ -88,18 +88,14 @@
...
-        <a class="p-button--positive js-invoke-modal" href="/openstack/contact-us?product=openstack">
-          Get in touch
-        </a>
+        <a class="p-button--positive" href="#get-your-openstack">Get OpenStack</a>
       </p>
       <p>
-        <a href="/engage/cloud-pricing-report-2021">
-          Read our cloud pricing report&nbsp;&rsaquo;
-        </a>
+        <a href="/engage/cloud-pricing-report-2021">Save up to two-thirds compared to hyperscalers&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/openstack/index.html:98]:
```suggestion
        <a href="/engage/cloud-pricing-report">Save up to two-thirds compared to hyperscalers&nbsp;&rsaquo;</a>
```

**Review by anthonydillon — APPROVED** [review]
Looks good with one suggestion in the code and could you change the "Charmed OpenStack" link at the bottom of the page to https://charmhub.io/openstack-base please.

**Pull request merged**